daniel_guerrero/flask_fundamentals/ninja/ninja.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, render_template, request, redirect
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+	return render_template('index.html')
+
+@app.route('/ninja')
+def ninja():
+	return render_template('ninja.html')
+
+@app.route('/ninja/blue')
```

Comment:
This is good what you have so far. I'd recommend trying to do this assignment with route parameters. You should be able to complete this assignment with only have three methods using route params. @app.route('/ninja/<color>')